dev/TeachingTip/TeachingTip.xaml

```diff
@@ -13,6 +13,7 @@
         <Setter Property="Background" Value="{ThemeResource TeachingTipBackgroundBrush}"/>
         <Setter Property="Foreground" Value="{ThemeResource TeachingTipForegroundBrush}"/>
         <Setter Property="BorderBrush" Value="{ThemeResource TeachingTipBorderBrush}"/>
+        <Setter Property="BorderThickness" Value="{ThemeResource TeachingTipContentBorderThicknessUntargeted}"/>
```

Comment:
Am I reading this right that BorderThickness only does something when a teaching tip is untarggetted? so setting the border thickness on a targetted teaching tip does nothing?

Reply:
Yes since we need to set the borderthickness to 0 on the side where the little arrow thingy for targeting is. Using the borderthickness for every case would create weird visuals since the excluded side would almost never line up.

Reply:
Besides that, there are lightweightstyling resources for all cases so retemplating is not needed to customize these points.